pkg/transaction/transaction_test.go

```diff
@@ -369,6 +385,9 @@ func TestTransactionSend(t *testing.T) {
 				backendmock.WithPendingNonceAtFunc(func(ctx context.Context, account common.Address) (uint64, error) {
 					return nonce, nil
 				}),
+				backendmock.WithSuggestGasTipCapFunc(func(ctx context.Context) (*big.Int, error) {
+					return big.NewInt(0), nil
```

Comment:
Possibly return a unique number across the tests that would be recognized as the SuggestedGasTipCap?